reactjs-films-homework/src/languages/getLanguage.ts

```diff
@@ -0,0 +1,26 @@
+import data from './languagesData';
+
+const lang = (language: string) => {
+  let globalLang;
+
+  switch (language) {
+    case '':
+      globalLang = data.EN;
+      break;
+    case 'EN':
+      globalLang = data.EN;
+      break;
+    case 'RU':
+      globalLang = data.RU;
+      break;
+    default:
+      globalLang = data.EN;
+      break;
+  }
+  return globalLang;
+};
+
+export const langArr = ['RU', 'EN'];
```

Comment:
It's a good practice to store such values in enums.
https://www.typescriptlang.org/docs/handbook/enums.html